src/main/java/org/spongepowered/mod/mixin/core/entity/player/MixinEntityPlayer.java

```diff
@@ -50,4 +67,48 @@ public EntityItem dropOneItem(boolean dropAll) {
     public EntityItem dropPlayerItemWithRandomChoice(ItemStack itemStackIn, boolean unused) {
         return this.dropItem(itemStackIn, false, false);
     }
+
+    @Overwrite
```

Comment:
Actually, Forge changes more than just firing the event - it adds methods like `isBed()` and `setBedOccupied()`.